src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -370,11 +447,130 @@ export function getGenericAssayEnrichmentRowData(
     });
 }
 
+export function getGenericAssayBinaryEnrichmentRowData(
+    genericAssayBinaryEnrichments: GenericAssayBinaryEnrichment[],
+    groups: { name: string; nameOfEnrichmentDirection?: string }[]
+): GenericAssayBinaryEnrichmentRow[] {
+    return genericAssayBinaryEnrichments.map(genericAssayBinaryEnrichment => {
+        let countsWithAlteredPercentage = _.map(
+            genericAssayBinaryEnrichment.counts,
+            datum => {
+                const alteredPercentage =
+                    datum.count > 0 && datum.totalCount > 0
+                        ? (datum.count / datum.totalCount) * 100
+                        : 0;
+                return {
+                    ...datum,
+                    alteredPercentage,
+                };
+            }
+        );
+        let enrichedGroup = '';
+        // fallback to stable id if name is not specified
+        let entityName =
```

Comment:
This is a suggestion for you, It's okay to leave it as is.

`let`: You should use this when you know the variable will likely change in the function
`const`: You should use this when you know the variable will not change in the function